src/main/java/com/mixpanel/android/util/HttpService.java

```diff
@@ -27,8 +27,36 @@
  * An HTTP utility class for internal use in the Mixpanel library. Not thread-safe.
  */
 public class HttpService implements RemoteService {
+
+    private static boolean sAdBlockerEnabled;
```

Comment:
Do you mind renaming `sAdBlockerEnabled` to something more specific? AdBlocker being enabled doesn't mean Mixpanel gets blocked.